src/tests/math/math_test_fp.c

```diff
@@ -0,0 +1,89 @@
+/**
+ * @math_test_fp
+ * @simple tests for fp functions in embox just to make sure the corresponding functions are not stubs
+ * @date 03.03.2020
+ * @author Rushil kvs
+ */
+
+#include <embox/test.h>
+#include <math.h>
+
+EMBOX_TEST_SUITE("Tests for fp functions");
+
+bool Equal(double a, double b)
+{
+	double epsilon = 0.00000000000001;
+	if ((a-b) < epsilon && -(a-b) < epsilon) return true;
+  	return false;
+}
+
+bool Equalf(float a, float b)
+{
+	float epsilon = 0.00000000001;
+	if ((a-b) < epsilon && -(a-b) < epsilon) return true;
+  	return false;
+}
+
+long double Equall(long double a, long double b){
+	long double epsilon = 0.0000000000000000000001;
+	if ((a-b) < epsilon && -(a-b) < epsilon) return true;
+  	return false;
+}
```

Comment:
As I understand, it is required because the results are not simple floating-point numbers like `1.5`, but `1.545463634634(8)`, right? Can we make such tests to only use simple floating-point numbers like `1.5`, `3.7` etc.? If os, then you can use simple `test_assert_equal` instead of these epsilons.

Reply:
I guess, we can remove Equalf and Equall. But we can't use simple test_assert_equal , as it directly checks a==b.

Reply:
So, we need to have atleast one Equal function

Reply:
Hmm, let's check this, I commented below for the `modf(2.34, &t)`.

Reply:
Ok, let's leave this one `equal` function.